### PR TITLE
restore lost commit and add escape symbol

### PR DESCRIPTION
**Pull request by vladimir-dudnik**

### CODE REVIEW

File: demos/action_recognition_demo/python/README.md [hunk 1]
```diff
@@ -130,13 +130,14 @@ python3 action_recognition_demo.py \
     -lb <omz_idr>/demos/action_recognition_demo/python/driver_actions.txt
 ```
 
+When a single image is applied as an input, the demo processes and renders it quickly, then exits. To continuously visualize processed results on the screen, apply the `loop` option, which enforces looping over processing a single image.
```
Comment (andrew-zaytsev) [inline demos/action_recognition_demo/python/README.md:133]: >**NOTE**: If you provide a single image as an input, the demo processes and renders it quickly, then exits. To continuously visualize inference results on the screen, apply the `loop` option, which enforces processing a single image in a loop.

File: demos/action_recognition_demo/python/README.md [hunk 2]
```diff
@@ -130,13 +130,14 @@ python3 action_recognition_demo.py \
...
 To avoid disk space overrun in case of continuous input stream, like camera, you can limit the amount of data stored in the output file(s) with the `limit` option. The default value is 1000. To change it, you can apply the `-limit N` option, where `N` is the number of frames to store.
 
->**NOTE**: Windows* systems may not have the Motion JPEG codec installed by default. If this is the case, OpenCV FFMPEG backend can be downloaded by the PowerShell script provided with the OpenVINO install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. Run the script with Administrative privileges. Alternatively, you can save results as images.
+>**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, OpenCV FFMPEG backend can be downloaded by the PowerShell script provided with the OpenVINO install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. Run the script with Administrative privileges. Alternatively, you can save results as images.
```
Comment (andrew-zaytsev) [inline demos/action_recognition_demo/python/README.md:140]: - OpenCV FFMPEG backend can be downloaded by the PowerShell script --> you can can download OpenCV FFMPEG back end using the PowerShell script
- the OpenVINO install package --> the OpenVINO™ toolkit install package, 
- Administrative privileges --> administrative privileges

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: demos/action_recognition_demo/python/README.md (hunk 1 above)
Comment (vladimir-dudnik) [reply demos/action_recognition_demo/python/README.md]: done

File: demos/action_recognition_demo/python/README.md (hunk 2 above)
Comment (vladimir-dudnik) [reply demos/action_recognition_demo/python/README.md]: changed

File: demos/action_recognition_demo/python/README.md [hunk 3]
```diff
@@ -130,14 +130,15 @@ python3 action_recognition_demo.py \
...
 To avoid disk space overrun in case of continuous input stream, like camera, you can limit the amount of data stored in the output file(s) with the `limit` option. The default value is 1000. To change it, you can apply the `-limit N` option, where `N` is the number of frames to store.
 
->**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, OpenCV FFMPEG backend can be downloaded by the PowerShell script provided with the OpenVINO install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. Run the script with Administrative privileges. Alternatively, you can save results as images.
+>**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, you can can download OpenCV FFMPEG back end using the PowerShell script, provided with the OpenVINO &trade; install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. The script should be run with administrative privileges if OpenVINO &trade; installed in system protected folder (this is a typical case). Alternatively, you can save results as images.
```
Comment (andrew-zaytsev) [inline demos/action_recognition_demo/python/README.md:141]: - Remove extra 'can': If this is the case, you can can download --> If this is the case, you can download
- Remove coma: using the PowerShell script, provided --> using the PowerShell script provided
- if OpenVINO &trade; installed in system protected folder --> if OpenVINO &trade; is installed in a system protected folder

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: demos/action_recognition_demo/python/README.md (hunk 3 above)
Comment (vladimir-dudnik) [reply demos/action_recognition_demo/python/README.md]: fixed

**Review by andrew-zaytsev — APPROVED** [review]

File: demos/action_recognition_demo/python/README.md [hunk 4]
```diff
@@ -130,13 +130,15 @@ python3 action_recognition_demo.py \
     -lb <omz_idr>/demos/action_recognition_demo/python/driver_actions.txt
 ```
 
+>**NOTE**: If you provide a single image as an input, the demo processes and renders it quickly, then exits. To continuously visualize inference results on the screen, apply the `loop` option, which enforces processing a single image in a loop.
```
Comment (eizamaliev) [inline demos/action_recognition_demo/python/README.md:133]: May be the `--loop` instead `loop`?

**Pull request merged**